urllib3/transport_security.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from six.moves.http_cookiejar import split_header_words
+
+__all__ = ['TransportSecurityManager', 'TransportSecurityStore']
+
+
+log = logging.getLogger(__name__)
```

Comment:
Let's remove this logger for now: I expect it to come back in in a future pull request, but that's ok.